sweeper.go

```diff
@@ -20,6 +22,14 @@ var alpha = 1 - math.Exp(-1.0)
 // The global sweeper.
 var globalSweeper sweeper
 
+var globalClock clock.Clock
+
+// SetClock sets a clock to use in the sweeper.
+// This will probably only ever be useful for testing purposes.
+func SetClock(cl clock.Clock) {
```

Comment:
Any reason not to keep the tests in the same package so we can make this private?

Reply:
We need this for the bandwidth tests in -core, see https://github.com/libp2p/go-libp2p-core/pull/276.